media/base/android/java/src/org/chromium/media/MediaPlayerBridge.java

```diff
@@ -78,22 +84,22 @@ protected void destroy() {
         mNativeMediaPlayerBridge = 0;
     }
 
-    protected MediaPlayer getLocalPlayer() {
-        if (mPlayer == null) {
-            mPlayer = new MediaPlayer();
+    protected ExMediaPlayer getExMediaPlayer() {
```

Comment:
No need to change name, getLocalPlayer will also be ok. 
Is there an easy way to switch from MediaPlayer to ExoMediaPlayer, , co-exist?